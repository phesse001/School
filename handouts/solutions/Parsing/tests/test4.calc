two = 1 + 1;
zero = two - two;
print zero;
