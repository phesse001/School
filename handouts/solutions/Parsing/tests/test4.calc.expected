Expr parsed as NUM!
Expr parsed as NUM!
Expr parsed as plus!
Stmt parsed as assignment!
Expr parsed as ID: two
Expr parsed as ID: two
Expr parsed as minus!
Stmt parsed as assignment!
Expr parsed as ID: zero
Stmt parsed as print!
Empty stmt list parsed!
Stmt list parsed!
Stmt list parsed!
Stmt list parsed!
Program parsed correctly!
Ignoring invalid parse tree for printing...
